code/game/objects/items/RCD.dm

```diff
@@ -982,6 +988,16 @@ RLD
 	else
 		create_machine(A, user)
 
+/obj/item/construction/plumbing/research
+	name = "research plumbing constructor"
+	desc = "A type of plumbing constructor designed to rapidly deploy the machines needed to conduct cytological research."
+	icon_state = "plumberer_sci"
+	has_ammobar = TRUE
+
+/obj/item/construction/plumbing/Initialize(mapload)
```

Comment:
You meant `/obj/item/construction/plumbing/research/Initialize(mapload)` here.
However, instead of assigning the variable twice, make it call a proc such as `set_plumbing_design_types()` which the child overrides.